### PR TITLE
add google analytics

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

**Comment by sebastianbenz** [comment]
@jeffjose can you please take a look whether I've setup things correctly
@matthiasrohmer please check if I wired things up correctly and if all templates now have an analytics integration.

File: boilerplate/templates/index.html [hunk 1]
```diff
@@ -272,23 +272,19 @@ <h3>Bookend Sample</h3>
          role="button"
          on="tap:boilerplate.scrollTo(duration=200)"
          class="fab">View Code</div>
-    <amp-analytics type="googleanalytics" id="analytics">
-      <script type="application/json">
-        {
-          "vars": {
-            "account": "UA-73836974-1"
-          },
-          "triggers": {
```
Comment (jeffjose) [inline boilerplate/templates/index.html]: Did we miss porting over `triggers` over to the new setup/

Comment (sebastianbenz) [reply boilerplate/templates/index.html]: I thought that this is not really needed as page views are tracked by default. Or am I wrong about this?

**Comment by matthiasrohmer** [comment]
LGTM at first glance. I'd like to take another look tomorrow with a clear mind. Something minor I thought about immediately is that we could disable tracking for local development and staging at least for all pages generated by Grow, if it helps. Would be something along the lines of

```jinja2
{% if not doc.pod.env.dev and not podspec.env.name == 'staging' %}
<amp-analytics ...>
```

**Comment by sebastianbenz** [comment]
Tracking is disabled for staging and localhost via GA.

File: boilerplate/templates/index.html (hunk 1 above)
Comment (sebastianbenz) [reply boilerplate/templates/index.html]: We discussed this offline.

**Pull request merged**